bin/methods/create_a_student.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../classes/student'
+
+class CreateNewStudent
+  def initialize(people)
+    @people = people
+  end
+
+  def create_a_student
+    clear
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp.downcase
+    new_student = Student.new(age, name, parent_permission == 'y')
+    @people << new_student
+    added_msg('Student')
+  end
```

Comment:
- [ ] good job cleaning up the main.rb file 👍 in this method text is printed in the console, information is gathered and it is processed, this is not in line with the single responsibility principle, try to split this into smaller methods with a single responsibility. for more details have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid).